doc/en/developer/source/policies/psc.rst

```diff
@@ -84,49 +84,30 @@ Process
-   
-   * +1 : For
-   * -1 : Against
-   * +0: Mildly for, but mostly indifferent
-   * -0: Mildly against, but mostly indifferent
-
- * A -1 vote must be accompanied with a detailed reason of being against.
- * A vote is *successful* if there is a majority of positive votes.
- * A vote is *unanimous*, if there are either:
+GeoServer Improvement Proposals
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-   #. No -1 votes against it, or
-   #. No +1 votes for it.
+A GeoServer Improvement Proposals (GSIPs) is needed for any action that:
 
- * In the event of an *successful non unanimous* vote, the following steps are taken:
- 
-   * Each member who votes -1 _may_ supply an alternative with which the original author can use to rework the proposal in order to satisfy that PSC member.
-   * If at least one -1 voting PSC member supplies some alternative criteria, the original author must rework the proposal and resubmit, and the voting process starts again from scratch.
-   * If no -1 voters are able to supply alternative criteria, the proposal is accepted.
-   * In the event of an *unsuccessful* vote, the author may rework and submit. A proposal may not be resubmitted after being rejected three times.
-   * Note that a majority of positive votes does not need to be a majority of the full PSC, just a majority of the 'active' PSC - defined by those present at an IRC meeting or responding within 4 days on email.  PSC members need not sound in on every single motion, but are expected to be active - the section on stepping down details that if a PSC is not active for 2 months they will
+  * Has a major effect on others in the GeoServer community; or
+  * Will break backwards compatibility; or
+  * Change core code
 
-When not to use the GSIP
-^^^^^^^^^^^^^^^^^^^^^^^^
+For more on making a proposal see :ref:`gsip`.
 
-A GSIP is only needed for:
+.. include:: gsip_voting.txt
 
-  * an action that has a major effect on others in the GeoServer community.
-  * If an action will break backwards compatibility, or change core code, a GSIP is recommended.
+Snap Decisions
+^^^^^^^^^^^^^^
 
-A GSIP is NOT needed for:
+A GSIP proposal is NOT needed for:
 
-  * an improvement that can go in a community module; or
-  * a bug fix that doesn't rework anything substantially
+* an improvement that can go in a community module; or
+* a bug fix that doesn't rework anything substantially
 
-For minor decisions where feedback might be desired, the course of action to take is to consult the development list or raise it in an irc meeting.  The GeoServer Project recognizes that it is run those who are actually doing the work, and thus we want to avoid high overhead for 'getting things done'.
+For minor decisions where feedback might be desired, the course of action to take is to consult the development list or raise it in a video meeting.  The GeoServer Project recognizes that it is run those who are actually doing the work, and thus we want to avoid high overhead for 'getting things done'.
```

Comment:
This probably needs some amendment. When was this written, the whole PSC was actively participating in meetings, nowadays only a portion takes part in the video calls. We can amend it simply by saying that members not participating to the meeting can follow up after the meeting minutes mail.

Reply:
I think that is already covered by the “developer list” being mentioned.

Reply:
(In both cases we end up with public communication on the dev list)

Reply:
(In both cases we end up with public communication on the dev list)